### PR TITLE
Start Forseti service automatically at boot

### PR DESCRIPTION
**Pull request by dlinares-linux**
In a VM install, if the server instance is restarted, forseti
service was disabled and never started again. This change allows
the forseti service to be automatically started at boot.

### CODE REVIEW

**Comment by dlinares-linux** [comment]
@googlebot I signed it!

**Comment by gkowalski-google** [comment]
The issue that this PR is trying to solve for is that in @dlinares-linux case, some of the Ubuntu security updates are failing to install and the startup script is exiting out without starting Forseti. Since these errors are happening early in the startup script, this change would not resolve the issue. Issue created #286

File: modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl [hunk 1]
```diff
@@ -136,6 +136,9 @@ sleep 5
 echo "Forseti Startup - Attempting to update database schema, if necessary."
 python3 $USER_HOME/forseti-security/install/gcp/upgrade_tools/db_migrator.py
 
+# Enable main Forseti service to start automatically at boot
+systemctl enable forseti
+
 # Start main Forseti service
 echo "Forseti Startup - Starting Forseti service."
 systemctl start forseti
```
[the remark this answers is not in the document]
Comment (natking1987) [reply modules/server/templates/scripts/forseti-server/forseti_server_startup_script.sh.tpl]: [](url)p

**Comment by kevensen** [comment]
@dlinares-linux @gkowalski-google Just FYI, I've updated this branch.  Once all the tests pass, I can merge this in.

**Pull request merged**